department-of-veterans-affairs/caseflow
# JHoang/APPEALS-23663

Resolves 23663

# Description
Fixed issue where no description was displayed on the Case Timeline after adding an issue from the Edit Issues page.

## Acceptance Criteria

## Testing Plan
1. Login as Intake Admin BVADWISE
2. Click on Mail Intake. Click Start Link
3. Select all the information required from the Decision Review Request form
4. Click on Continue to Next step
5. Click on Add Issue button and add an issue (with MST/PACT)
6. Establish the appeal
7. Click on Correct Issue link from the Case Details Page
8. Click on Add Issue link >> Add issue and click on Save.
9. Verify Issue description under "Update task completed" section on Case Timeline.

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
